### ISSUE TITLE
Dead links in the readme

### ISSUE THREAD

**Issue by jkeen** [shipshapecode/tether#462]
The readme links to https://github.hubspot.com/tether/ for documentation, and those links are dead.

Not sure if you're managing http://tether.io/ now, too, but that site is loading weird too.

**Comment by rwwagner90** [comment]
Definitely need to fix these. Not entirely sure how though.

**Comment by TrevorBurnham** [comment]
> Not sure if you're managing http://tether.io/ now, but that site is loading weird too.

It looks OK from here. A network blip, perhaps?

**Comment by rwwagner90** [comment]
@TrevorBurnham it's fully broken with a lot of 404. I think github.hubspot.com was perhaps removed?

**Comment by TrevorBurnham** [comment]
😕 Weird, https://github.hubspot.com/ loads just fine for me too…

**Comment by rwwagner90** [comment]
@TrevorBurnham hmm, I think we mean https://github.hubspot.com/tether/ specifically.

**Comment by TrevorBurnham** [comment]
Looks like you already solved it: 69fa6af

**Comment by rwwagner90** [comment]
@TrevorBurnham no, not solved yet. No idea how to solve all the 404s from http://tether.io/

There are a bunch of resources that are set to `github.hubspot.com` still and I don't know where those are coming from.

<img width="1371" alt="Screen Shot 2020-02-26 at 8 41 34 PM" src="https://user-images.githubusercontent.com/2640861/75404393-67732100-58d8-11ea-8ebb-322150e9d59c.png">

**Comment by TrevorBurnham** [comment]
> There are a bunch of resources that are set to `github.hubspot.com` still and I don't know where those are coming from.

That domain is hard-coded into the Jade templates on the gh-pages branch, which all the HTML files are built from: https://github.com/shipshapecode/tether/blob/76e57602e477072a659024feb781211edf62396f/resources/templates/base.jade#L18-L34

**Comment by rwwagner90** [comment]
Ah thanks @TrevorBurnham! I did not realize those templates existed. That fixed the 404 issue, but now I think we're just hitting a lot of issues with Tether changes. Will try to fix it all soon.

**Comment by Desan** [comment]
@rwwagner90 it's still broken :thinking:

**Comment by rwwagner90** [comment]
@Desan yeah, haven't been able to figure out how to fix it

**Comment by rwwagner90** [comment]
@TrevorBurnham any idea how hsdoc works? I keep trying to edit templates and building overwrites my changes.

**Comment by TrevorBurnham** [comment]
@rwwagner90 Hmm… sorry, it's a mystery to me too!

**Comment by rwwagner90** [comment]
@TrevorBurnham any idea who we could ask?

**Comment by TrevorBurnham** [comment]
^ @zackbloom @adamschwartz Does this bring back memories for you? 😄

**Comment by zackbloom** [comment]
A trip down memory road! hsdoc, as you can imagine, is a build tool on top of (I think?) grunt. Can we just change the domain in the built files like little cheaters, or do we need the build to work?

**Comment by rwwagner90** [comment]
@zackbloom the hsdoc command seems to blow away all my changes every time. I'm trying to update `base.jade` but if I delete these two lines, they just come back when I run `hsdoc -a` https://github.com/shipshapecode/tether/blob/gh-pages/resources/templates/base.jade#L45-L46

I guess rather than using the jade templates we have defined here, hsdoc somehow has a default template setup that adds these things back again?

**Comment by zackbloom** [comment]
I wish I knew the answer, but I don't even have a copy of the hsdoc source anymore. The smart thing would probably be to port it to a more modern build chain, but I can't say how much work that would be.

**Comment by tilgovi** [comment]
Looks like tether.io maybe expired and got swooped up by the Tether crytocurrency project. It now redirects to tether.to.
